charts/fybrik/files/taxonomy/adminrules.json

```diff
@@ -75,24 +75,55 @@
         }
       }
     },
+    "RangeType": {
+      "type": "object",
+      "properties": {
+        "max": {
+          "type": "integer"
+        },
+        "min": {
+          "type": "integer"
+        }
+      }
+    },
     "Restriction": {
       "type": "object",
-      "additionalProperties": {
-        "$ref": "#/definitions/StringList"
+      "required": [
+        "property"
+      ],
+      "properties": {
+        "property": {
+          "type": "string"
+        },
+        "range": {
+          "$ref": "#/definitions/RangeType"
```

Comment:
Range should be optional, right?

Reply:
It is optional (does not appear in the 'required' section, line 91)